app/vmui/packages/vmui/src/utils/default-server-url.ts

```diff
@@ -1,12 +1,16 @@
 import { getAppModeParams } from "./app-mode";
 import { replaceTenantId } from "./tenants";
-const { REACT_APP_LOGS } = process.env;
+import { AppType } from "../types/appType";
+import { getFromStorage } from "./storage";
+const { REACT_APP_TYPE } = process.env;
 
 export const getDefaultServer = (tenantId?: string): string => {
   const { serverURL } = getAppModeParams();
+  const storageURL = getFromStorage("SERVER_URL") as string;
```

Comment:
could it be defined as a const
```"SERVER_URL"```

Reply:
The function `getFromStorage` takes strings specified in `StorageKeys`. I think it's redundant to extract the key into a separate variable. 
An alternative could be to change `StorageKeys` to an enum, but in such a case, there will be many unnecessary imports.